projects/ion/src/lib/sidebar/sidebar.component.ts

```diff
@@ -14,8 +14,35 @@ export class IonSidebarComponent {
 
   public closed = true;
 
+  public checkClikOnPageAccess = (
+    event
+  ): EventListenerOrEventListenerObject => {
+    this.checkClikOnPage(event);
+    return;
+  };
+
+  public checkClikOnPage(event): EventListenerOrEventListenerObject {
```

Comment:
Essa método retorna `undefined`. Então, você pode tipar o retorno como `void` e remover o `return`.

```suggestion
  public checkClikOnPage(event): void {
```